client/state/index.js

```diff
@@ -111,7 +112,11 @@ export const reducer = combineReducers( {
 	wordads,
 } );
 
-const middleware = [ thunkMiddleware, noticesMiddleware ];
+const middleware = [
+	thunkMiddleware,
+	noticesMiddleware,
+	navigationMiddleware,
```

Comment:
Fixed by 4c46e748f2228622fd3b51980b94e371c34962e9 ... _maybe_ . It is now complaining about this to me:

```
Uncaught (in promise) TypeError: Cannot redefine property: state
    at Function.defineProperty (<anonymous>)
    at eval (index.js:125)
    at eval (enhancer.js:36)
    at vendor.development.js?v=147ad937b5:43322
    at createReduxStore (index.js:307)
    at loadInitialStateFailed (initial-state.js:107)
```
It looks like [consoleDispatcher](https://github.com/Automattic/wp-calypso/blob/master/client/state/console-dispatch/index.js#L72) got called twice after this change. I have no idea why :(

@dmsnell Could you help me take a look?